x-pack/plugins/observability/server/saved_objects/slo.ts

```diff
@@ -53,6 +67,7 @@ export const slo: SavedObjectsType = {
       budgetingMethod: { type: 'keyword' },
       enabled: { type: 'boolean' },
       tags: { type: 'keyword' },
+      version: { type: 'long' },
```

Comment:
i guess for version it may make sense since we want to filter by version. Out of curiosity we are we mapping other fields in SLO definition in saved object like indicator etc

Reply:
That's legacy reason. Before we have the summary index, all our searches was going through the saved object repository. So we had to map these fields. Now they are not necessary, but removing saved object mapping is forbidden I believe.